Cargo.toml

```diff
@@ -80,3 +80,7 @@ rustdoc-args = ["--cfg", "docsrs"]
 
 [package.metadata.playground]
 features = ["ws"]
+
+[[example]]
+name = "chat"
+required-features = ["ws"]
```

Comment:
Whoa! I didn't realize you can do this. Thats cool! We should totally add for the other examples as well. Doesn't have to be done in this PR though.